tests/optional_chaining/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -128,11 +128,8 @@ foo.bar.baz
 =====================================output=====================================
 function foo() {
   return (
-    a
-      .b()
-      .c()
-      // Comment
-      ?.d()
+    a.b().c()// Comment
+    ?.d()
```

Comment:
Should this still be broken onto multiple lines? This doesn’t look very good, especially the missing space before the inline comment.

Reply:
Ah, thought I had addressed this. Will push a fix shortly

Reply:
Seems like it’s still happening?

Reply:
Yeah, as I said I missed it. Thought I'd fixed but must have been looking in the wrong spot. I'm doing this outside of work so will @ you when it's fixed.

Reply:
@j-f1 I did some digging, and it might be that this PR has unearthed a pre-existing issue, since the same bug can be reproduced by just changing the break length from 3 to 4: https://github.com/prettier/prettier/compare/master...mmkal:optional-chaining-comment-bug . That said, I'll look into this more anyway - if it wasn't possible before and now is, it's a regression even if the code causing it has always been there.

Reply:
@j-f1 now (actually) fixed.